en/intel-rsd.md

```diff
@@ -1,19 +1,24 @@
 Title: Intel RSD | MAAS
-TODO:  Need access to this hardware to produce quality docs
+TODO:  Need access to this hardware to produce quality docs (Newell Jensen is a resource)
+table_of_contents: True
 
 
 # Intel RSD
 
 Intel Rack Scale Design (RSD) is a hardware architecture that allows the
 dynamic composition of physical systems from a pool of available hardware
-resources.
+resources. It is an example of *composable hardware*.
```

Comment:
My only suggestion is that we should define composable hardware somewhere. Perhaps a link to a new Concept/term?